<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>ARDrone Parameters:</string>
    </property>
    <widget class="QPushButton" name="bt_takeoff_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Take Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_landing_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Landing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_emergency_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Emergency</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_Disconnect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_indicator">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>59</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_connect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Follow Mode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ball">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Ball</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_battery_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>FlatTrim</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Magnetometer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <widget class="QCheckBox" name="checkBox_cameraF">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera Frontal</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cameraD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera Inferior</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_th_image">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>400</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Vision Algorithms</string>
    </property>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>YUV</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_hsv_segmentation">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV Segmentation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Segmentation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Kalman Filter</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_filter">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Noise Filter</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_filter1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_filter2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_filter3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_filter4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter 4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_adi">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>252</width>
      <height>252</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_image">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_compass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>252</width>
      <height>252</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
